README.md

```diff
@@ -2,11 +2,11 @@ Welcome to the official documentation for Linaro and the Reference Software Plat
 
 ## Contents
 
-- [Reference Platform Home](Reference-Platform/README.md)
+- [Reference Platform Home](https://github.com/Linaro/documentation/blob/master/Reference-Platform/README.md)
```

Comment:
That isn't right.

We should be converting README.md into index.html and then just referencing `Reference-Platform`.